client/src/components/application/Stats/Stats.vue

```diff
@@ -80,7 +80,7 @@ export default {
       requests: null,
       numRecords: null,
       currentPage: 1,
-      perPage: 0,
+      perPage: 100,
```

Comment:
This was set to 0 because the documentation for the b-pagination says it should be set to 0 for when the api handles the paging.  We set it to  > 0 if we want the UI to handle the paging internally.  Not sure what the impact of this will be if we're getting exactly 100 per page from the api.

Reply:
Hmm, yes when it was set to zero I only ever got 1 page link. Was it the per-page attribute on the b-table that was supposed to be left as zero? I can see that. But the page attribute on the actual b-pagination I changed to the variable since it was hardcoded 100. And the perPage data value as zero was sending to the API as zero.